Support all i18n keys for messages_ru.properties

petrov9 opened the pull request:
Now all messages_ru.properties files a full copies of a related messages_en.properties file but with translated values

Closes #40336

petrov9 committed:
Support all i18n keys for messages_ru.properties

Now all messages_ru.properties file is a full copy of a related messages_en.properties file but with translated values

Closes #40336

Signed-off-by: Anton Petrov <petrov9810@gmail.com>

petrov9 commented:
@ahus1 solved conflicts with Weblate, there was some problem with branch, so created a new PR

ahus1 commented:
@petrov9 - you might have been looking for this functionality: https://docs.weblate.org/en/latest/user/files.html

After you select the language and the component, you should find the menu entries. If you don't find them, I can double-check if it is a permission issue.

petrov9 commented:
> @petrov9 - you might have been looking for this functionality: https://docs.weblate.org/en/latest/user/files.html
> 
> After you select the language and the component, you should find the menu entries. If you don't find them, I can double-check if it is a permission issue.

Yes, I can only download, but not to upload translations

ahus1 commented:
@petrov9 - can you please check that you selected the language and the component? 

Please navigate to https://hosted.weblate.org/projects/keycloak/account-ui/ru/ and try again. 
I just double checked with a second user account without extra permissions, and I see both upload and download. 

![image](https://github.com/user-attachments/assets/d5bfc06e-9c83-436b-a94c-fdb4f611bbe3)

petrov9 commented:
@ahus1 you are right, I had to choose language and component, but only choosing language didn't work, now I see this tab

pasternake commented on js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ru.properties:
```diff
@@ -14,6 +14,7 @@ credentialCreatedAt=<0>Создан</0> {{date}}.
 error-username-invalid-character='{{0}}' содержит недопустимый символ.
 usernamePlaceholder=Имя пользователя или email
 infoMessage=Нажав кнопку Удалить доступ, вы удалите предоставленные разрешения для этого приложения. Это приложение больше не будет использовать вашу информацию.
+scopesHelp=Области, связанные с этим ресурсом.
```
Мне с первого раза непривычно "области", привычнее скоупы, но это скорее слэнговое выражение. ИИ советует "области")

petrov9 replied:
Да, я тоже тут думал как лучше сделать, в итоге пообщавшись с ии пришли к мнение что scope-область, realm-realm

pasternake commented on js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ru.properties:
```diff
@@ -155,3 +180,39 @@ unLinkSuccess=Учетная запись успешно отсоединена
+addressScopeConsentText=Адрес
+phoneScopeConsentText=Номер телефона
+offlineAccessScopeConsentText=Оффлайн доступ
+samlRoleListScopeConsentText=Мои роли
```
Как вариант - Мои роли (SAML)

pasternake commented on js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ru.properties:
```diff
@@ -155,3 +180,39 @@ unLinkSuccess=Учетная запись успешно отсоединена
+samlRoleListScopeConsentText=Мои роли
+rolesScopeConsentText=Роли пользователей
+organizations=Организации
+organizationDescription=Просмотр организаций, к которым вы присоединились
```
"Описание организации" ?

petrov9 replied:
organizationDescription=View organizations that you joined тут именно просмотр, а не описание в оригинале

pasternake commented on themes/src/main/resources-community/theme/base/admin/messages/messages_ru.properties:
```diff
@@ -1,22 +1,76 @@
+error-invalid-length-too-short=Атрибут {0} должен иметь минимальную длину {1}.
+error-invalid-length-too-long=Атрибут {0} должен иметь максимальную длину {2}.
+error-invalid-email=Неверный адрес электронной почты.
+error-invalid-number=Неверный номер.
```
Может быть по контексту - "Неверное число"

petrov9 replied:
Ага, поправил

pasternake commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -1,6 +1,17 @@
 emailVerificationBody=Кто-то создал учетную запись {2} с этим E-mail. Если это были Вы, нажмите на следующую ссылку для подтверждения вашего email\n\n{0}\n\nЭта ссылка устареет через {3}.\n\nЕсли Вы не создавали учетную запись, просто проигнорируйте это письмо.
 emailVerificationBodyHtml=<p>Кто-то создал учетную запись {2} с этим E-mail. Если это были Вы, нажмите по ссылке для подтверждения вашего E-mail</p><p><a href="{0}">Ссылка для верификации e-mail</a></p><p>Эта ссылка устареет через {3}.</p><p>Если Вы не создавали учетную запись, просто проигнорируйте это письмо.</p>
+orgInviteSubject=Приглашение присоединиться к организации {0}
+orgInviteBody=Вас пригласили присоединиться к организации "{3}". Нажмите на ссылку ниже, чтобы присоединиться.\n\n{0}\n\nЭта ссылка истекает в течение {4}.\n\nЕсли вы не хотите присоединяться к организации, просто проигнорируйте это сообщение.
```
вы -> Вы

pasternake commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -1,6 +1,17 @@
+orgInviteSubject=Приглашение присоединиться к организации {0}
+orgInviteBody=Вас пригласили присоединиться к организации "{3}". Нажмите на ссылку ниже, чтобы присоединиться.\n\n{0}\n\nЭта ссылка истекает в течение {4}.\n\nЕсли вы не хотите присоединяться к организации, просто проигнорируйте это сообщение.
+orgInviteBodyHtml=<p>Вас пригласили присоединиться к организации {3}. Нажмите на ссылку ниже, чтобы присоединиться.</p><p> <a href="{0}">Ссылка для присоединения к организации</a></p><p> Срок действия этой ссылки истекает через {4}.</p><p> Если вы не хотите вступать в организацию, просто проигнорируйте это сообщение.</p>
+orgInviteBodyPersonalized=Привет, "{5}" "{6}".\n\n Вас пригласили присоединиться к организации "{3}". Нажмите на ссылку ниже, чтобы присоединиться.\n\n{0}\n\nЭта ссылка истекает в течение {4}.\n\nЕсли вы не хотите присоединяться к организации, просто проигнорируйте это сообщение.
```
вы -> Вы

pasternake commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -1,6 +1,17 @@
+orgInviteBodyPersonalized=Привет, "{5}" "{6}".\n\n Вас пригласили присоединиться к организации "{3}". Нажмите на ссылку ниже, чтобы присоединиться.\n\n{0}\n\nЭта ссылка истекает в течение {4}.\n\nЕсли вы не хотите присоединяться к организации, просто проигнорируйте это сообщение.
+orgInviteBodyPersonalizedHtml=<p>Привет, {5} {6}.</p><p> Вас пригласили присоединиться к организации {3}. Нажмите на ссылку ниже, чтобы присоединиться.</p><p> <a href="{0}">Ссылка для присоединения к организации</a></p><p> Срок действия этой ссылки истекает через {4}.</p><p> Если вы не хотите вступать в организацию, просто проигнорируйте это сообщение.</p>
+emailUpdateConfirmationSubject=Подтвердите новый адрес электронной почты
+emailUpdateConfirmationBody=Чтобы обновить свою учетную запись {2} с адресом электронной почты {1}, нажмите на ссылку ниже\n\n{0}\n\nСрок действия этой ссылки истекает в течение {3}.\n\nЕсли вы не хотите продолжать это изменение, просто проигнорируйте это сообщение.
```
вы -> Вы

petrov9 replied:
Сделал

pasternake commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -1,6 +1,17 @@
+orgInviteBodyPersonalizedHtml=<p>Привет, {5} {6}.</p><p> Вас пригласили присоединиться к организации {3}. Нажмите на ссылку ниже, чтобы присоединиться.</p><p> <a href="{0}">Ссылка для присоединения к организации</a></p><p> Срок действия этой ссылки истекает через {4}.</p><p> Если вы не хотите вступать в организацию, просто проигнорируйте это сообщение.</p>
+emailUpdateConfirmationSubject=Подтвердите новый адрес электронной почты
+emailUpdateConfirmationBody=Чтобы обновить свою учетную запись {2} с адресом электронной почты {1}, нажмите на ссылку ниже\n\n{0}\n\nСрок действия этой ссылки истекает в течение {3}.\n\nЕсли вы не хотите продолжать это изменение, просто проигнорируйте это сообщение.
+emailUpdateConfirmationBodyHtml=<p>Чтобы обновить свою учетную запись {2} с адресом электронной почты {1}, нажмите на ссылку ниже</p><p> <a href="{0}">Ссылка для обновления e-mail</a></p><p> Срок действия этой ссылки истекает через {3}.</p><p> Если вы не хотите продолжать вносить изменения, просто проигнорируйте это сообщение.</p>
```
вы -> Вы

pasternake commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -9,7 +20,7 @@ passwordResetBody=Кто-то только что запросил измене
 executeActionsSubject=Обновление Вашей учетной записи
 executeActionsBody=Администратор просит Вас обновить данные Вашей учетной записи {2}, выполнив следующие действия: {3} . Нажмите по следующей ссылке чтобы начать этот процесс.\n\n{0}\n\nЭта ссылка устареет через {4}.\n\nЕсли у вас есть подозрения, что администратор не мог сделать такой запрос, просто проигнорируйте это письмо.
-executeActionsBodyHtml=<p>Администратор просит Вас обновить данные Вашей учетной записи {2}, выполнив следующие действия: {3}. Нажмите по следующей ссылке чтобы начать этот процесс.</p><p><a href="{0}">Ссылка для обновления аккаунта</a></p><p>Эта ссылка устареет через {4} минут.</p><p>Если у вас есть подозрения, что администратор не мог сделать такой запрос, просто проигнорируйте это письмо.</p>
+executeActionsBodyHtml=<p>Администратор просит Вас обновить данные Вашей учетной записи {2}, выполнив следующие действия: {3}. Нажмите по следующей ссылке чтобы начать этот процесс.</p><p><a href="{0}">Ссылка для обновления аккаунта</a></p><p>Эта ссылка устареет через {4}.</p><p>Если у вас есть подозрения, что администратор не мог сделать такой запрос, просто проигнорируйте это письмо.</p>
```
у вас -> у Вас

pasternake commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -22,3 +33,34 @@ eventUpdatePasswordBodyHtml=<p>Ваш пароль был изменен в {0}
 eventUpdateTotpBody=OTP был обновлен в вашей учетной записи {0} с {1}. Если это были не Вы, пожалуйста, свяжитесь с администратором.
 eventUpdateTotpBodyHtml=<p>OTP был обновлен в вашей учетной записи {0} с {1}. Если это были не Вы, пожалуйста, свяжитесь с администратором.</p>
+eventUpdateCredentialSubject=Обновить учетные данные
+eventUpdateCredentialBody=Ваши учетные данные {0} были изменены {1} с {2}. Если это были не вы, обратитесь к администратору.
```
вы -> Вы

pasternake commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -22,3 +33,34 @@ eventUpdatePasswordBodyHtml=<p>Ваш пароль был изменен в {0}
+eventUserDisabledByTemporaryLockoutHtml=<p>Ваш пользователь временно отключен из-за нескольких неудачных попыток на {0}. При необходимости обратитесь к администратору.</p>
+eventUserDisabledByPermanentLockoutSubject=Пользователь отключен постоянной блокировкой
+eventUserDisabledByPermanentLockoutBody=Ваш пользователь был отключен навсегда из-за нескольких неудачных попыток на {0}. Обратитесь к администратору.
+eventUserDisabledByPermanentLockoutHtml=<p>Ваш пользователь был отключен навсегда из-за нескольких неудачных попыток на {0}. Обратитесь к администратору.</p>
```
"Ваш пользователь" - непонятен контекст

petrov9 replied:
Сделал

petrov9 commented on js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ru.properties:
```diff
@@ -155,3 +180,39 @@ unLinkSuccess=Учетная запись успешно отсоединена
+webauthn-passwordless-help-text=Используйте свой ключ доступа для входа без пароля.
+passwordless=Без пароля
+error-invalid-multivalued-size=Атрибут {{0}} должен иметь не менее {{1}} и не более {{2}} значений.
+recovery-authn-code=Мои коды аутентификации восстановления
```
Ага, тут тоже recovery-authn-code=My recovery authentication codes

petrov9 committed:
Support all i18n keys for messages_ru.properties. Update translations

Now all messages_ru.properties file is a full copy of a related messages_en.properties file but with translated values

Closes #40336

Signed-off-by: Anton Petrov <petrov9810@gmail.com>

petrov9 committed:
Support all i18n keys for messages_ru.properties. Add keycloak.v2/admin translation

Now all messages_ru.properties file is a full copy of a related messages_en.properties file but with translated values

Closes #40336

Signed-off-by: Anton Petrov <petrov9810@gmail.com>

petrov9 commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -22,3 +33,34 @@ eventUpdatePasswordBodyHtml=<p>Ваш пароль был изменен в {0}
+eventUpdateCredentialBodyHtml=<p>Ваши учетные данные {0} были изменены {1} с {2}. Если это были не вы, обратитесь к администратору.</p>
+eventRemoveCredentialSubject=Удалить учетные данные
+eventRemoveCredentialBody=Учетные данные {0} были удалены из вашего аккаунта {1} из {2}. Если это были не вы, обратитесь к администратору.
+eventRemoveCredentialBodyHtml=<p>Учетные данные {0} были удалены из вашего аккаунта {1} из {2}. Если это были не вы, обратитесь к администратору.</p>
```
Сделал

petrov9 commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -22,3 +33,34 @@ eventUpdatePasswordBodyHtml=<p>Ваш пароль был изменен в {0}
+eventRemoveCredentialBodyHtml=<p>Учетные данные {0} были удалены из вашего аккаунта {1} из {2}. Если это были не вы, обратитесь к администратору.</p>
+eventUserDisabledByTemporaryLockoutSubject=Пользователь отключен из-за временной блокировки
+eventUserDisabledByTemporaryLockoutBody=Ваш пользователь временно отключен из-за нескольких неудачных попыток на {0}. При необходимости обратитесь к администратору.
+eventUserDisabledByTemporaryLockoutHtml=<p>Ваш пользователь временно отключен из-за нескольких неудачных попыток на {0}. При необходимости обратитесь к администратору.</p>
```
Сделал

petrov9 commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -22,3 +33,34 @@ eventUpdatePasswordBodyHtml=<p>Ваш пароль был изменен в {0}
+eventUserDisabledByTemporaryLockoutBody=Ваш пользователь временно отключен из-за нескольких неудачных попыток на {0}. При необходимости обратитесь к администратору.
+eventUserDisabledByTemporaryLockoutHtml=<p>Ваш пользователь временно отключен из-за нескольких неудачных попыток на {0}. При необходимости обратитесь к администратору.</p>
+eventUserDisabledByPermanentLockoutSubject=Пользователь отключен постоянной блокировкой
+eventUserDisabledByPermanentLockoutBody=Ваш пользователь был отключен навсегда из-за нескольких неудачных попыток на {0}. Обратитесь к администратору.
```
Сделал

petrov9 commented:
@pasternake я еще добавил файл keycloak.v2/admin

petrov9 committed:
Support all i18n keys for messages_ru.properties. Add more translations

Now all messages_ru.properties file is a full copy of a related messages_en.properties file but with translated values

Closes #40336

Signed-off-by: Anton Petrov <petrov9810@gmail.com>

petrov9 commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -1,6 +1,17 @@
 emailVerificationBodyHtml=<p>Кто-то создал учетную запись {2} с этим E-mail. Если это были Вы, нажмите по ссылке для подтверждения вашего E-mail</p><p><a href="{0}">Ссылка для верификации e-mail</a></p><p>Эта ссылка устареет через {3}.</p><p>Если Вы не создавали учетную запись, просто проигнорируйте это письмо.</p>
+orgInviteSubject=Приглашение присоединиться к организации {0}
+orgInviteBody=Вас пригласили присоединиться к организации "{3}". Нажмите на ссылку ниже, чтобы присоединиться.\n\n{0}\n\nЭта ссылка истекает в течение {4}.\n\nЕсли вы не хотите присоединяться к организации, просто проигнорируйте это сообщение.
+orgInviteBodyHtml=<p>Вас пригласили присоединиться к организации {3}. Нажмите на ссылку ниже, чтобы присоединиться.</p><p> <a href="{0}">Ссылка для присоединения к организации</a></p><p> Срок действия этой ссылки истекает через {4}.</p><p> Если вы не хотите вступать в организацию, просто проигнорируйте это сообщение.</p>
```
Сделал

petrov9 commented on themes/src/main/resources-community/theme/base/email/messages/messages_ru.properties:
```diff
@@ -1,6 +1,17 @@
+orgInviteBody=Вас пригласили присоединиться к организации "{3}". Нажмите на ссылку ниже, чтобы присоединиться.\n\n{0}\n\nЭта ссылка истекает в течение {4}.\n\nЕсли вы не хотите присоединяться к организации, просто проигнорируйте это сообщение.
+orgInviteBodyHtml=<p>Вас пригласили присоединиться к организации {3}. Нажмите на ссылку ниже, чтобы присоединиться.</p><p> <a href="{0}">Ссылка для присоединения к организации</a></p><p> Срок действия этой ссылки истекает через {4}.</p><p> Если вы не хотите вступать в организацию, просто проигнорируйте это сообщение.</p>
+orgInviteBodyPersonalized=Привет, "{5}" "{6}".\n\n Вас пригласили присоединиться к организации "{3}". Нажмите на ссылку ниже, чтобы присоединиться.\n\n{0}\n\nЭта ссылка истекает в течение {4}.\n\nЕсли вы не хотите присоединяться к организации, просто проигнорируйте это сообщение.
+orgInviteBodyPersonalizedHtml=<p>Привет, {5} {6}.</p><p> Вас пригласили присоединиться к организации {3}. Нажмите на ссылку ниже, чтобы присоединиться.</p><p> <a href="{0}">Ссылка для присоединения к организации</a></p><p> Срок действия этой ссылки истекает через {4}.</p><p> Если вы не хотите вступать в организацию, просто проигнорируйте это сообщение.</p>
```
Сделал

pasternake reviewed (approved):
Готово!

ahus1 commented:
@petrov9 - I see that @pasternake approved, but added some more comments/suggestions. 

How to plan to proceed?

petrov9 commented on js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_ru.properties:
```diff
@@ -1,600 +1,3497 @@
+groupDeleted_other=Группы удалены
+acrToLoAMappingHelp=Определите, какое значение ACR (Authentication Context Class Reference) сопоставляется с каким LoA (Level of Authentication). ACR может иметь любое значение, тогда как LoA должен быть числовым.
+uploadFile=Загрузить JSON-файл
+loginActionTimeoutHelp=Макс. время, которое пользователь имеет для выполнения действий, связанных с входом в систему, таких как обновление пароля или настройка TOTP. Реком
```
Здесь используется or

petrov9 committed:
Support all i18n keys for messages_ru.properties. Translate clarification

Now all messages_ru.properties file is a full copy of a related messages_en.properties file but with translated values

Closes #40336

Signed-off-by: Anton Petrov <petrov9810@gmail.com>

petrov9 commented on js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_ru.properties:
```diff
@@ -1,600 +1,3497 @@
+groupDeleted_other=Группы удалены
+acrToLoAMappingHelp=Определите, какое значение ACR (Authentication Context Class Reference) сопоставляется с каким LoA (Level of Authentication). ACR может иметь любое значение, тогда как LoA должен быть числовым.
+uploadFile=Загрузить JSON-файл
+loginActionTimeoutHelp=Макс. время, которое пользователь имеет для выполнения действий, связанных с входом в систему, таких как обновление пароля или настройка TOTP. Реком
```
Перевел как клейм

petrov9 commented:
@ahus1 I added changes that @pasternake suggested, so now everything is ready

ahus1 committed:
Review

Signed-off-by: Alexander Schwartz <alexander.schwartz@gmx.net>

ahus1 reviewed (approved):
Thank you for the translation and the review, this is a big step forward for the RU translation!
